### PR TITLE
[CHORE] adding prometheus handler

### PR DESCRIPTION
**Pull request by nicolastakashi**
## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #2357

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [X] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

File: mindsdb/integrations/handlers/prometheus_handler/prometheus_handler.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+        self.handler.call_prometheus_api(params)
+        return None
+
+class PrometheusHandler(APIHandler):
+    """A class for handling connections and interactions with the Prometheus API.
+
+    Attributes:
+        bearer_token (str): The consumer key for the Prometheus app.
+        api (tweepy.API): The `tweepy.API` object for interacting with the Prometheus API.
```
Comment (Smyja) [inline mindsdb/integrations/handlers/prometheus_handler/prometheus_handler.py:53]: why is this here?

File: mindsdb/integrations/handlers/prometheus_handler/README.md
Comment (Smyja) [inline mindsdb/integrations/handlers/prometheus_handler/README.md:1]: create an example on how to use this. Like the Jira integration.

**Comment by ZoranPandovski** [comment]
Hi @Smyja can you please provide more information once you review other users' code instead of just saying `why is this here`? This will help the maintainers for review but also contributors so they can improve the code

**Comment by ZoranPandovski** [comment]
Thanks @nicolastakashi we will review this. So far, amazing work

**Comment by ZoranPandovski** [comment]
Hey @nicolastakashi can you please provide more info please? How did you build it without testing? Please check https://docs.mindsdb.com/contribute/app-handlers. If you are not sure how to run the SQL commands it should be same as any other integration by just providing the required parameters for Prometheus and using the `prometeus` as engine.

**Comment by nicolastakashi** [comment]
@ZoranPandovski actually I need help to run minds locally to test it, I have no clue how to do this.
There's any guide how to run the project locally to test it?

**Comment by ZoranPandovski** [comment]
https://docs.mindsdb.com/contribute/install

**Comment by AMK9978** [comment]
Altogether, as Prometheus is super high in-demand, I'd like to work on this issue.

**Comment by nicolastakashi** [comment]
Hi @AMK9978 I would love to help, but I didn't manage to test it locally, could you help with this?

**Comment by AMK9978** [comment]
Sure! I'm gonna test it. Later, we need to provide anomalies forecasting in real-time.

**Comment by ZoranPandovski** [comment]
Once tested please comment so we can re-open this. Thanks
